core/src/main/scala/scala/macros/core/Universe.scala

```diff
@@ -26,12 +24,15 @@ trait Universe extends Gensym {
   type PatVar // NOTE(olafur) subject for removal
   type Defn
 
+  def fresh(prefix: String): String
```

Comment:
Would it make sense to replace this with `TermNameFresh: TermName`?

Reply:
Seems it complicates the service protocol a little bit.